### ISSUE TITLE
Add Site wide background image to export 

### ISSUE THREAD

**Issue by bph** [WordPress/create-block-theme#568]
With Gutenberg 18.1 comes [side wide background image](https://make.wordpress.org/core/2024/04/10/whats-new-in-gutenberg-18-1-10-april/#set-site-background-images-in-the-site-editor) feature. 

The set image should be part of the theme export

**Comment by bph** [comment]
~~Actually, I am not sure if that's actually necessary.~~  It turns out when the theme is uploaded to a site that has the Gutenberg plugin enabled (18.1) then the site-wide background image shows out of the box.  I just can't see the image file in any folder of the downloaded theme. 
~~It's the most curious thing. ~~

**Comment by bph** [comment]
Of course, it showed up because the LocalWP site where I uploaded the background image was still active.   🤦‍♀️

It's hard-coded in the theme.json 
```
"background": {		
        "backgroundImage": {
	"id": 195,
	"source": "file",
	"title": "3032452",
	"url": "http://localhost:10115/wp-content/uploads/2024/04/3032452-scaled.jpg"
	},
	"backgroundPosition": "28.000000000000004% 59%",
	"backgroundSize": "auto"
```

**Comment by pbking** [comment]
We have added the [theme-ization of SOME blocks such as image and cover block](https://github.com/WordPress/create-block-theme/blob/trunk/admin/create-theme/theme-media.php#L137).  We haven't added anything to theme.json yet though and this seems like it would be the best first candidate.  It would be very similar and not terribly difficult.

**Comment by bph** [comment]
Hmm, I am not sure what your comments is getting at. And it seems I didn't make myself clear enought. 

**A side wide background image is not at a block level like an image or a cover block.** 

<img width="1202" alt="Screenshot 2024-04-18 at 16 21 27" src="https://github.com/WordPress/create-block-theme/assets/39980/dc71b3da-d73e-4b85-b32f-11de2abc961e">
It influences the whole site and it a theme relevant new feature in the site editor. 
The export already puts the high-level settings into in the theme.json. 

## What did I expect?
That the background image was:
- downloaded into the /assets folder like fonts are downloaded in the /fonts folder and 
- the location in the theme.json is adopted to the relative location in the asset folder. 

{
"background": {		
        "backgroundImage": {
	"id": 195,
	"source": "file",
	"title": "3032452",
	"url": "../assets/2024/04/3032452-scaled.jpg"
	},
	"backgroundPosition": "28.000000000000004% 59%",
	"backgroundSize": "auto"
}

## How to test it: 
- Needs Gutenberg 18.1.x
- Create a blank theme
- Add a background image to the site via Site Editor > Styles > Style Icon >  see screenshot below for location. 
- Save 
- Save it in CBT 
- Download in zip
- delete the background image from the media library
- upload the zip file to a different site. 
- You will notice that the background image is not showing on the new site. The reason is that hardcoded link to the existing site in the theme.json file.

<img width="222" alt="Screenshot 2024-04-19 at 11 30 34" src="https://github.com/WordPress/create-block-theme/assets/39980/c8bf06ed-9795-4b8b-834c-a0881236f4ea">

**Comment by pbking** [comment]
> Hmm, I am not sure what your comments is getting at. And it seems I didn't make myself clear enought.
A side wide background image is not at a block level like an image or a cover block.

I was just noting that the type of processing we do for blocks (download a linked asset to a local folder, changing the source to point to the now theme-local asset) is the same processing we would do for the sitewide background image in theme.json.  We haven't done any type of processing like that on theme.json but it's very similar to what we are already doing for blocks and shouldn't be very difficult.

**Comment by mikachan** [comment]
Related PR in Gutenberg: https://github.com/WordPress/gutenberg/pull/60578
